### PR TITLE
Exit CLI

### PR DESCRIPTION
**Pull request by m1ga**
https://github.com/kefranabg/readme-md-generator/issues/149

### CODE REVIEW

**Comment by m1ga** [comment]
now linter is happy :smile:

File: src/cli.js [hunk 1]
```diff
@@ -34,4 +34,6 @@ module.exports = async ({ customTemplatePath, useDefaultAnswers }) => {
   await readme.writeReadme(readmeContent)
 
   utils.showEndMessage()
+
+  yargs.exit(0)
```
Comment (kefranabg) [inline src/cli.js]: Doing an exit is a bit rough. Actually I found the bug, it's a yargs regression, the solution is just to fix the version to 14.0.0

**Comment by kefranabg** [comment]
Doing an exit is a bit rough. Actually I found the bug, it's a yargs regression, the solution is just to fix the version to 14.0.0

**Comment by m1ga** [comment]
I see, that's even a better way

**Pull request closed without merging**